tests/shell.nix

```diff
@@ -7,6 +7,8 @@ testLib.runTests {
       custom_nodejs = symlinkJoin {
         name = "custom-nodejs";
         paths = [ nodejs ];
+        version = "12.8.3";
```

Comment:
What is this version string used for? Is it the nodejs version we are using?

Reply:
https://github.com/tweag/npmlock2nix/blob/github-experiment/internal.nix#L191

The test won't work without this because it is used by the `nodeSource` function.